### PR TITLE
Translate README

### LINKED ISSUE #185: Translate README
Project is finished, README should be translated to English

### PR DESCRIPTION
**Pull request by hjaremko**
Close #185

### CODE REVIEW

**Review by lsereda — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,79 +1,81 @@
...
-#### Rozgrywka
+#### Gameplay
 
-Gra jest w pełni tekstowa i zainspirowana gatunkiem *Single-User Dungeon* (np. Otchłań).
-Gracz dokonuje iterakcji ze światem poprzez wpisywanie odpowiednich poleceń,
-takich jak `go`, `investigate` lub `pick`. Pełny spis możliwych akcji znajduje
-się w sekcji *Dokumentacja*.
+The game is using a text-user-interface inspired by the Single-User Dungeon genre *(ex. Otchłań (pol))*.
+The player interacts with the world by typing in proper commands such as `go`, `investigate` or `pick`.
+Full description of available command can be found in the *Documentation* section.
```
Comment (seqre) [inline README.md:13]:
```suggestion
Full list of available commands is in the *Documentation* section.
```

File: README.md [hunk 2]
```diff
@@ -1,79 +1,81 @@
...
-rozmowa, walka), zbierania przedmiotów.
-1. Na każdym etapie gry istnieją różne możliwości dalszego przebiegu rozgrywki.
-1. Zebrane przedmioty gracz może wykorzystać w dalszym przebiegu gry (w zależności od przedmiotu do
-walki, obrony lub zwiększenia danego atrybutu).
-1. Wraz z pokonywaniem kolejnych przeszkód gracz będzie zdobywał doświadczenie rozwijając poszczególne atrybuty.
+The game flow is as follows:
+1. First, the player picks his character's class.
+1. Every class offers unique abilities.
+1. The player goes through successive locations where he has the opportunity to explore and interact with other characters
+(e.g. conversation, fight), collecting items.
```
Comment (seqre) [inline README.md:20]:
```suggestion
1. The player goes through successive locations where he has the opportunity to explore, interact with other characters (e.g. conversation, fight), and collect items.
```
I'm also not sure whether there should be a comma before where

File: README.md [hunk 3]
```diff
@@ -1,79 +1,81 @@
...
+1. At each stage of the game, there are different possible courses of the game.
+1. The collected items can be used in the further course of the game
+(depending on the item: for combat, defense, or increasing a given attribute)
```
Comment (seqre) [inline README.md:23]:
```suggestion
(depending on the item: for combat, defense, or to increase a given attribute)
```

File: README.md [hunk 4]
```diff
@@ -1,79 +1,81 @@
...
+### Building
 ```
 ./gradlew build -x test
 ```
-Wymaga ustawienia zmiennych środowiskowych zgodnie z opisem na [wiki](https://github.com/hjaremko/cute-animals/wiki/Konfiguracja-zmiennych-%C5%9Brodowiskowych).
+#### Required environment variables
+`SPRING_DATASOURCE_URL` : `jdbc:postgresql://localhost:5432/<database name>`  
+`SPRING_DATASOURCE_USERNAME` : `postgres`  
+`SPRING_DATASOURCE_PASSWORD` : your password  
+`SPRING_PROFILES_ACTIVE` : `dev`
```
Comment (seqre) [inline README.md:51]: I think we should point out there is `dev` and `prod` environment

File: README.md [hunk 5]
```diff
@@ -1,79 +1,81 @@
...
 
-### Dokumentacja
+### Documentation
 
-Pełną dokumentację, raporty testów i pokrycia oraz plik `jar`
-można znaleźć w [artefaktach z ostatniego buildu](https://github.com/hjaremko/cute-animals/actions).
 
-#### Diagramy klas znajdują się [tutaj](./assets/UML.md).
+Full documentation, test and coverage reports, binary files
+can be found in [artifacts](https://github.com/hjaremko/cute-animals/actions).
```
Comment (seqre) [inline README.md:57]:
```suggestion
are in [artifacts](https://github.com/hjaremko/cute-animals/actions).
```

File: README.md [hunk 6]
```diff
@@ -1,79 +1,81 @@
...
+- ```investigate``` - provides a description of the current location  
+- ```talk <character_name>``` - allows you to talk to individual characters
+- ```go <location_name>``` - allows you to go to another location  
+- ```pick <item_name>``` - allows you to pick up an item and put it in your backpack  
+- ```throw <item_name>``` - allows you to discard an item out of the backpack  
+- ```equip <item_name>``` - allows you to put on an item from a backpack  
+- ```off <item_name>```- allows you to unequip an item and put it in the backpack  
+- ```backpack``` - lists the current state of the backpack  
+- ```eq``` - lists the currently equipped items  
+- ```stats``` - lists character statistics
```
Comment (seqre) [inline README.md:73]:
```suggestion
- ```stats``` - lists the character's statistics  
```
For consistency's sake

File: README.md [hunk 7]
```diff
@@ -1,79 +1,81 @@
...
+- ```pick <item_name>``` - allows you to pick up an item and put it in your backpack  
+- ```throw <item_name>``` - allows you to discard an item out of the backpack  
+- ```equip <item_name>``` - allows you to put on an item from a backpack  
+- ```off <item_name>```- allows you to unequip an item and put it in the backpack  
+- ```backpack``` - lists the current state of the backpack  
+- ```eq``` - lists the currently equipped items  
+- ```stats``` - lists character statistics  
+- ```skills``` - lists the character's abilities  
+- ```fight <character_name>``` - allows you to fight individual characters  
+- ```attack``` - takes a turn in combat mode
```
Comment (seqre) [inline README.md:76]:
```suggestion
- ```attack``` - allows you to attack in combat mode 
```

**Review by seqre — CHANGES_REQUESTED** [review]

**Review by seqre — APPROVED** [review]
LGTM

**Pull request merged**